CRM/Contribute/ActionMapping/ByType.php

```diff
@@ -209,7 +209,7 @@ public function createQuery($schedule, $phase, $defaultParams) {
         ->param('selectedStatuses', $selectedStatuses);
     }
 
-    if ($schedule->recipient_listing && $schedule->limit_to) {
+    if ($schedule->recipient_listing && in_array($schedule->limit_to), [2, 3]) {
```

Comment:
in_array for comparing two values could just be replaced by an OR ?

($schedule->limit_to === 2 || $schedule->limit_to== 3])

Reply:
@agileware-justin actually I prefer in_array over multiple expressions with OR or && for readability and faster, (more detail [here](https://stackoverflow.com/questions/324665/which-is-faster-in-array-or-a-bunch-of-expressions-in-php)) although the speed performance is negligible. Is there a reason `in_array` would cause issue?